### ISSUE TITLE
Breadcrumb: wrong css display settings for icons

### ISSUE THREAD

**Issue by morvael** [primefaces/primefaces#4959]
## 1) Environment
- PrimeFaces version: 7.0.4
- Application server + version: WildFly 16.0.0
- Affected browsers: all

## 2) Expected behavior

Icon displayed alongside label.

## 3) Actual behavior

Label displayed under icon:
![icon-block](https://user-images.githubusercontent.com/1133953/60517720-728d9700-9ce0-11e9-9c5b-144a4cb69b7f.png)

`.ui-breadcrumb ul li span` and `.ui-icon` applies `display:block` to icon span.

In regular menu this works correctly because lower-level `.ui-menu .ui-menuitem-link .ui-menuitem-icon, .ui-menu .ui-submenu-icon` applies `display:inline-block` to icon span.

**Comment by tandraschko** [comment]
Please provide a sample xhtml and which theme(s) are effected.

**Comment by morvael** [comment]
My workday has now ended, so it will be ready in about 18 hours. I think problem appears in more than one theme, perhaps in all of them. Simply put Breadcrumb isn't icon friendly at this time.

**Comment by morvael** [comment]
Every theme is affected. Attached is an xhtml page which allows to observe this in various themes:
[breadcrumb.zip](https://github.com/primefaces/primefaces/files/3353749/breadcrumb.zip)

According to IDE hints default iconPos is on the right. It doesn't work this way:
![bugs](https://user-images.githubusercontent.com/1133953/60572559-268d3180-9d76-11e9-935c-c1f30b1d5816.png)

The same xhtml allows to check the other [bug](https://github.com/primefaces/primefaces/issues/4958), so I will attach it there as well.

**Comment by mertsincan** [comment]
I'll fix it on the next elite versions and 7.1. For now, please use;
```
body .ui-breadcrumb ul li span {
    display: inline;
    margin-right: .3em;`
}
```
